lib/CL/pocl_img_buf_cpy.c

```diff
@@ -0,0 +1,199 @@
+/* pocl_img_buf_cpy.c: common parts of image and buffer copying
```

Comment:
Pls check the style in this file especially. http://portablecl.org/docs/html/development.html#coding-style